src/modelinstance.cpp

```diff
@@ -732,10 +751,46 @@ Status ModelInstance::loadOVModelUsingCustomLoader() {
     }
     return StatusCode::OK;
 }
-
+}  // namespace ovms
+namespace ovms {
 void ModelInstance::loadCompiledModelPtr(const plugin_config_t& pluginConfig) {
     OV_LOGGER("ov::Core: {}, ov::Model: {}, targetDevice: {}, ieCore.compile_model(model, targetDevice, pluginConfig", reinterpret_cast<void*>(&ieCore), reinterpret_cast<void*>(this->model.get()), this->targetDevice);
-    compiledModel = std::make_shared<ov::CompiledModel>(ieCore.compile_model(this->model, this->targetDevice, pluginConfig));
+    if (this->targetDevice.find("GPU") != std::string::npos) {
+        if (false) {  // use this only for test for now until we have config management
+            cl_platform_id platformId;
+            cl_device_id deviceId;
+            auto oclContext = get_cl_context(platformId, deviceId);  // TODO use params from config, use device from config
+            this->oclContextC = oclContext;
+            this->oclContextCpp = std::make_unique<ov::intel_gpu::ocl::ClContext>(this->ieCore, this->oclContextC);
+            ov::intel_gpu::ocl::ClContext ov_context(this->ieCore, this->oclContextC);
+            OV_LOGGER("ov::Core: {} compile_model(model: {}, target_device:{}, pluginConfig:{})", (void*)&this->ieCore, (void*)&this->model, this->targetDevice, (void*)&pluginConfig);
```

Comment:
`(void*)&this->model` will log pointer of shared_ptr object instead of underlying model object, I do not think you want that here, use `(void*)this->mode.get()`